kernel/sysfs.c

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <myst/eraise.h>
+#include <myst/hostfile.h>
+
+#define SYS_DEVICE_CPU_DIR "/sys/devices/system/cpu"
+#define SYS_DEVICE_NODE_DIR "/sys/devices/system/node"
+
+int sysfs_setup()
+{
+    int ret = 0;
+    ECHECK(myst_copy_host_directory_recursively(
```

Comment:
Consider mounting a ramfs based filesystem at /sys vs adding directly to the rootfs. That way we won't write back in the case of hostfs.